generation/WinSDK/enums.json

```diff
@@ -8235,8 +8235,60 @@
   },
   {
     "addUsesTo": "DVASPECT",
-    "members": [],
+    "members": [
+      {
+        "name": "DVASPECT_OPAQUE "
```

Comment:
Remove the extra space.

Reply:
I don't think the new members are being added. I guess I have to combine the enums by defining it manually?